keps/sig-node/2902-cpumanager-distribute-cpus-policy-option/README.md

```diff
@@ -249,9 +311,23 @@ This delay should be minimal.
 
 No, the algorithm will run on a single `goroutine` with minimal memory requirements.
 
+### Troubleshooting
+
+###### How does this feature react if the API server and/or etcd is unavailable?
+
+No impact. The behavior of the feature does not change when API Server and/or etcd is unavailable since the feature is node local.
+
+###### What are other known failure modes?
+
+No known failure modes.
```

Comment:
worth mentioning perhaps that a possible failure mode is that because of the current CPU resource fragmentation, a distributed allocation can be impossible, while a packed allocation can be possible.

But IIRC this failure mode is automatically mitigated by the current implementation (and design? need to check this very KEP) which automatically falls back to packed mode when distributed mode is impossible.

In hindsight, we should record this behavior in the KEP if not done already

Reply:
That's correct, we always fallback to packed CPU allocation if resources are fragmented such that it is not possible to distribute CPUs. E.g. If all Available CPUs are present on the same NUMA node. 

This is already captured in the design details section of the KEP.

I will add this as a possible failure mode and in this section.